src/django/api/processing.py

```diff
@@ -245,30 +272,30 @@ def get_automatic_match_result(list_item_id, facility_id, confidence, context,
     return result
 
 
-def get_potential_match_result(list_item, item, items_count, context,
-                               should_create, result):
+def get_potential_match_result(match_item, items_count, context,
+                               should_create):
     from api.serializers import FacilityIndexDetailsSerializer
 
     AUTOMATIC_THRESHOLD = 0.8
 
-    facility = Facility.objects.get(id=item.facility_id)
+    facility = Facility.objects.get(id=match_item.facility_id)
     facility_index = FacilityIndex.objects.get(id=facility.id)
     facility_dict = FacilityIndexDetailsSerializer(
         facility_index,
         context=context
         ).data
-    facility_dict['confidence'] = item.confidence
-
-    if item.confidence < AUTOMATIC_THRESHOLD or items_count > 1:
-        if should_create:
-            facility_dict['confirm_match_url'] = reverse(
-                'facility-match-confirm',
-                kwargs={'pk': item.id})
-            facility_dict['reject_match_url'] = reverse(
-                'facility-match-reject',
-                kwargs={'pk': item.id})
-    result['matches'].append(facility_dict)
```

Comment:
```
result['matches'].append(facility_dict)
result['item_id'] = list_item.id
result['status'] = list_item.status
```
This is removed because we have to perform this operation out of the loop of match items.

Reply:
It has been replaced into `handle_potential_matches`